src/extension/extension.ts

```diff
@@ -24,10 +24,32 @@ console.log("RudderStack keys:", Object.keys(Analytics));
 
 const WRITE_KEY = "2q3zybBJRd9ErKIpkTRSdIahQ0C";
 const DATA_PLANE_URL = "https://getbruinbumlky.dataplane.rudderstack.com";
+
+const ensureAutoLockEnabled = async () => {
+  const config: vscode.WorkspaceConfiguration = workspace.getConfiguration("workbench.editor");
+  const autoLockGroups: Record<string, boolean> = config.get("autoLockGroups") || {};
+
+  // Check if the setting needs to be updated
+  if (!autoLockGroups["mainThreadWebview-markdown.preview"]) {
```

Comment:
Would it be possible to extract it to a constant, and maybe use string interpolation to bind it to the components that enclose `mainThreadWebview` and `markdown.preview`?

That way we could mutate those components without having to keep this in sync with them.